examples/with-iron-session/lib/fetch/index.js

```diff
@@ -0,0 +1,27 @@
+// node-fetch has a browser entry that re-exports window.fetch
+// we're not using isomorphic-unfect because we do not need to support IE11
```

Comment:
Let's remove this comments, Next.js polyfills `fetch` for the browser, `isomorphic-unfetch` is never required. only a server side `fetch` lib like `node-fetch` or `@zeit/fetch` is required for API routes and server side sections of the page (like the code inside `getStaticProps`)

Reply:
Updated and used @zeit/fetch